app/views/flower_shops/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1><%= @flower_shops.name %></h1>
+<p>Shop Rating: <%= @flower_shops.rating %></p>
+<p>Same Day Delivery: <%= @flower_shops.same_day_delivery %></p>
```

Comment:
Are these files being re committed?

Reply:
Hmm @Jaybraum do they look different from yours at all? I am wondering if they're showing up here because I merged your changes into this branch? Not honestly sure.